.changelog/3746.txt

```diff
@@ -0,0 +1,4 @@
+```release-note:improvement
+install/runner: Allow additional arguments for `waypoint runner agent` command
+to be supplied to `waypoint runner install` CLI
+```
```

Comment:
Oh you know what, want to add a quick changelog about how we don't show that internal-only label to users in the CLI output?

Reply:
The changelog is going to be longer than the PR 😂